libs/user/src/lib/auth/guard/no-auth.guard.ts

```diff
@@ -2,21 +2,19 @@ import { Injectable } from '@angular/core';
 import { AuthService } from '../+state';
 import { map, } from 'rxjs/operators';
 import { OrganizationService } from '@blockframes/organization/+state/organization.service';
-import { getOrgModuleAccess, getCurrentApp, App } from '@blockframes/utils/apps';
-import { RouterQuery } from '@datorama/akita-ng-router-store';
+import { getOrgModuleAccess, App } from '@blockframes/utils/apps';
 import { combineLatest } from 'rxjs';
-import { CanActivate, Router } from '@angular/router';
+import { ActivatedRouteSnapshot, CanActivate, Router } from '@angular/router';
 
 @Injectable({ providedIn: 'root' })
 export class NoAuthGuard implements CanActivate {
   constructor(
     private authService: AuthService,
     private orgService: OrganizationService,
-    private routerQuery: RouterQuery,
     private router: Router,
-  ) {}
+  ) { }
 
-  canActivate() {
+  canActivate(activatedRoute: ActivatedRouteSnapshot) {
```

Comment:
call the params `next`
```suggestion
  canActivate(next: ActivatedRouteSnapshot) {
```